lib/generators/godmin/resource/resource_generator.rb

```diff
@@ -2,12 +2,13 @@
 
 class Godmin::ResourceGenerator < Godmin::Generators::NamedBase
   argument :attributes, type: :array, default: [], banner: "attribute attribute"
-
+  class_option :"skip-navigation", :type => :boolean, :default => false, :description => "Skips extending navigation partial with new link"
```

Comment:
Wups my bad... Old habits die slow. Of course, but won't have time until next Tuesday. Sorry for the delay